packages/components/src/theme/theme_default_dark_semantic.css

```diff
@@ -0,0 +1,72 @@
+/* stylelint-disable alpha-value-notation */
+:root {
+  /* General */
+  --dh-color-black: var(--dh-color-gray-50);
+  --dh-color-white: var(--dh-color-gray-800);
+  --dh-color-accent: var(--dh-color-blue-700);
+  --dh-color-border: var(--dh-color-gray-500);
+  --dh-color-background: var(--dh-color-black);
+  --dh-color-foreground: var(--dh-color-white);
+  --dh-color-grid-background: var(--dh-color-background);
+  --dh-color-content-background: var(--dh-color-gray-100);
+
+  /* Text */
+  --dh-color-text: var(--dh-color-gray-800);
+  --dh-color-text-highlight: rgba(76 125 238 / 0.3);
```

Comment:
Base theme shouldn't have any hard coded colors. These should be derived from the base color palette. Also should make it easy to change the base color they are based on. e.g. A client may prefer a green based highlight over blue based.